package.json

```diff
@@ -10,10 +10,10 @@
     "serve": "node cli/index.js & vue-cli-service serve",
     "build": "vue-cli-service build",
     "build:docs": "npm run build && node ./cli export ./docs -c tailwind.config.sample.js",
-    "prepublishOnly": "npm run lint --no-fix && npm run build",
     "lint": "vue-cli-service lint",
     "test": "echo \"No test specified\"",
-    "version": "npm run build:docs && git add docs"
+    "version": "npm run build:docs && git add docs",
```

Comment:
@mauliksoni110 Hey. What was the issue/bug that this commit fixes?

Reply:
@rogden that was something I was trying to install the repo locally from Github.
I see that you have updated the logic in the latest release(s), thanks!
We can disregard this PR.